src/App.jsx

```diff
@@ -14,45 +17,130 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [listOfGoods, setListOfGoods] = useState([...goodsFromServer]);
+  const [targetInnerText, setTargetInnerText] = useState('');
+  const [isReverse, setIsReverse] = useState(false);
+
+  const sortList = (list, type) => {
+    switch (type) {
+      case 'Sort alphabetically':
+        if (isReverse) {
+          return list.sort((good1, good2) => good2.localeCompare(good1));
+        }
+
+        return list.sort((good1, good2) => good1.localeCompare(good2));
+
+      case 'Sort by length':
+        if (isReverse) {
+          return list.sort((good1, good2) => good2.length - good1.length);
+        }
+
+        return list.sort((good1, good2) => good1.length - good2.length);
+
+      case 'Reverse':
+        return list.reverse();
+
+      default:
+        return list;
```

Comment:
let's rewrite your sort method.
create a switch with `sort alphabetically` and `sort by length` options.
and after that check, if it is reverse - return reversed array